### PR TITLE
APP-92 Fixed the access_checker.py, warcserver.py

### PR DESCRIPTION
**Pull request by omerijaz27**

### CODE REVIEW

**Review by mijho — COMMENTED** [review]
@omerijaz27 it's probably better to get either @Leeeebs or @mdncs to take a look over the python side.

I'd also remove the wombat submodule from the commit.

File: pywb/warcserver/access_checker.py [hunk 1]
```diff
@@ -145,39 +148,54 @@ def find_access_rule(self, url, ts=None, urlkey=None):
...
-                 }
+        if self.mw_takedowns:
+            access_response = url_blocked(
+                takedown_collection={
+                    "assume_role": os.getenv('ASSUME_ROLE'),
+                    "table_value": os.getenv('TABLE_VALUE'),
+                    "display_name": os.getenv('DISPLAY_NAME'),
+                    "table_region": os.getenv('TABLE_REGION'),
+                    "customer_slug": os.getenv('CUSTOMER_SLUG')
+                }, url=url)
```
Comment (Leeeebs) [inline pywb/warcserver/access_checker.py:159]: just a note, this doesn't take into account dates.

**Review by Leeeebs — CHANGES_REQUESTED** [review]
Need to remove the wombat stuff - but should be fine to release for testing

**Review by omgoo — CHANGES_REQUESTED** [review]
Now merged to TNA branch

**Pull request closed without merging**